database/storeprocds/spFindTransactionType.sql

```diff
@@ -0,0 +1,26 @@
+-- =============================================
+-- Create basic stored procedure template
+-- =============================================
+
+-- Drop stored procedure if it already exists
+IF EXISTS (
+  SELECT * 
+    FROM INFORMATION_SCHEMA.ROUTINES 
+   WHERE SPECIFIC_SCHEMA = N'dbo'
+     AND SPECIFIC_NAME = N'spFindTransactionType' 
+)
+   DROP PROCEDURE dbo.spFindTransactionType
+GO
+
+CREATE PROCEDURE dbo.spFindTransactionType
+	@RecId bigint
+AS
+	SELECT Top 1 RecId, Name, Description FROM TRANSTYPE
```

Comment:
Please use caps on keywords so, Top should be TOP and if table field is same as sys reserved keyword so write like this [Name], [Description]

Reply:
fixed